main.go

```diff
@@ -76,7 +77,9 @@ const (
 	LeaderLockName                 = "modelmesh-controller-leader-lock"
 	LeaderForLifeLockName          = "modelmesh-controller-leader-for-life-lock"
 	EnableInferenceServiceEnvVar   = "ENABLE_ISVC_WATCH"
+	EnableClusterSrvngRntmEnvVar   = "ENABLE_CSR_WATCH"
```

Comment:
Better to just use full words?
```suggestion
	EnableClusterServingRuntimeEnvVar   = "ENABLE_CSR_WATCH"
```